src/content/containers/Root.js

```diff
@@ -77,8 +84,20 @@ class ProfileViewWhenReadyImpl extends PureComponent {
           </div>
         );
       }
-      case 'FATAL_ERROR':
-        return <div>{"Couldn't load the profile from the store."}</div>;
+      case 'FATAL_ERROR': {
+        const message = ERROR_MESSAGES[dataSource] || "Couldn't retrieve the profile.";
+        let additionalMessage = null;
+        if (view.error) {
+          additionalMessage = `Error was "${view.error}". Please look the Web Console to find more information.`;
```

Comment:
There's not all that more information to be gained from the web console. Can we make this just `additionalMessage = view.error.toString();`?